superset_config.py

```diff
@@ -29,4 +29,15 @@
 AUTH_USER_REGISTRATION = True
 
 # The default user self registration role
-AUTH_USER_REGISTRATION_ROLE = "Gamma"
\ No newline at end of file
+AUTH_USER_REGISTRATION_ROLE = "Gamma"
+
+# Any other additional roles to be assigned to the user on top of the base role
+# Note: by design we cannot use AUTH_USER_REGISTRATION_ROLE to
+# specify more than one role
+AUTH_USER_ADDITIONAL_ROLES = ["sql_lab"]
```

Comment:
Is `Gamma` not a subset of `sql_lab`?

Reply:
I don't know as well, but the behaviour seemed as if it's so. You can find that by comparing the permissions for each role under Edit Role where role permissions are listed in your local superset env.

Reply:
https://superset.apache.org/docs/security/#:~:text=The%20sql_lab%20role%20grants%20access%20to%20SQL%20Lab.%20Note%20that%20while%20Admin%20users%20have%20access%20to%20all%20databases%20by%20default%2C%20both%20Alpha%20and%20Gamma%20users%20need%20to%20be%20given%20access%20on%20a%20per%20database%20basis.

Reply:
Great, thanks for digging.